bitnami/jaeger/templates/collector/deployment.yaml

```diff
@@ -0,0 +1,182 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name:  {{ include "jaeger.collector.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+  replicas: {{ .Values.collector.replicaCount }}
+  {{- if .Values.collector.updateStrategy }}
+  strategy: {{- toYaml .Values.collector.updateStrategy | nindent 4 }}
+  {{- end }}
+  template:
+    metadata:
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: query
```

Comment:
Same here, change the `component` to the correct one or the services will map the incorrect pods

Reply:
Ditto https://github.com/bitnami/charts/pull/13480#discussion_r1034653364